util/EFSOI_Utilities/src/gridinfo_efsoi.f90

```diff
@@ -0,0 +1,461 @@
+module gridinfo_efsoi
```

Comment:
Is this really necessary?
This is a big duplication of the code under `enkf`. We should be able to link to the `enkf` library for almost all of the needs that are needed.

Reply:
I'll look into this. It appeared from comments and documentation that Dave intended to use as much of the enkf code as possible, so there might reasons for the EFSOI versions, and revisiting their necessity was on my todo list.

Reply:
Do we need this? What does it do other than print a header at execution (which is incorrect).
We have been copy/pasting stuff without really understanding the need for it for a while.

Reply:
removed